docs/user-guide/iam-sso.md

```diff
@@ -0,0 +1,101 @@
+---
+title: SSO Configuration
+description: Learn how to integrate SSO authentication within the IOMETE Data Plane
+last_update:
+  date: 03/26/2024
+  author: Vugar Dadalov
+---
+
+import { Trash } from "@phosphor-icons/react";
+import Img from '@site/src/components/Img';
+
+IOMETE offers an interface to configure your SSO for authentication.
+
+---
+
+## Configuration
+
+To view SSO configuration page, navigate to the `Settings` menu item and switch to the `Single Sign-On` tab under `IAM`.
+
+<Img src="/img/user-guide/sso/sso-landing.png" alt="LDAP configuration" />
+
```

Comment:
I'd split this page in 2 section as wel

1. Section for SMAL
2. Section for OIDC